### PR TITLE
feat: make table row inline

### PR DESCRIPTION
**Pull request by keita-determined**
## Description
- I talked to @alexjdetermined about pinned experiments, and have decided that all rows should have the same height to achieve pin feature. In order to do it, I made table row inline
- Its for #4925

### Screenshot

https://user-images.githubusercontent.com/109113616/188760128-c725e48e-96df-4a57-a492-c4eeb8e41704.mov

## Test Plan
- Check if all rows are inline in table and no issue with it in experiment table
- Check if experiment table view height and row height are fixed as well

## Commentary (optional)
- I replaced `InlineEditor` with `Editor modal` in experiment table to do inline.

## Checklist
- [ ] Changes have been manually QA'd
- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.
- [ ] If modifying `/webui/react/src/shared/` verify `make -C webui/react test-shared` passes.

[DET-1234]: https://determinedai.atlassian.net/browse/DET-1234?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by EmilyBonar — CHANGES_REQUESTED** [review]
Looking on Netlify (using Firefox) the description field for one of the rows is overflowing and the row beneath it is 139px tall, more than 2x the 60px height you're trying to set

File: webui/react/src/components/InteractiveTable.module.scss [hunk 1]
```diff
@@ -81,7 +90,17 @@
...
   .rightClickableCellWrapper {
-    display: inline;
+    display: inline-table;
+    overflow-x: auto;
+    overflow-y: hidden;
+    width: 100%;
+  }
+  /* stylelint-disable at-rule-empty-line-before */
+  // only for firefox
+  @-moz-document url-prefix() {
```
Comment (keita-determined) [inline webui/react/src/components/InteractiveTable.module.scss:100]: Firefox doesn't work like Chrome or Safari, so I did a special treat for Firefox.

**Review by EmilyBonar — APPROVED** [review]
LGTM

File: webui/react/src/pages/ProjectDetails.tsx [hunk 2]
```diff
@@ -894,6 +894,7 @@ const ProjectDetails: React.FC = () => {
               preserveSelectedRowKeys: true,
               selectedRowKeys: settings.row ?? [],
             }}
+            scroll={{ y: `calc(100vh - ${availableBatchActions.length === 0 ? '230' : '280'}px)` }}
```
Comment (keita-determined) [inline webui/react/src/pages/ProjectDetails.tsx:897]: make the table height slightly shorter for Firefox. 🔥🦊

**Pull request merged**